dwave/plugins/torch/boltzmann_machine.py

```diff
@@ -133,15 +133,21 @@ def _setup_hidden(self):
         """Preprocess some indexes to enable vectorized computation of effective fields of hidden
         units."""
         self._connected_hidden = any(
-            a in self.hidden_nodes and b in self.hidden_nodes for a, b in self.edges)
+            a in self.hidden_nodes and b in self.hidden_nodes for a, b in self.edges
+        )
```

Comment:
I understand that this is Black's doing, right? Let's not auto-format unrelated files to the PR.